### ISSUE TITLE
Command exception when running /pv admin open command for offline players

### ISSUE THREAD

**Issue by ChiefWarPig** [unisteven/PlayerVaults#3]
Hi I am running a Paper 121.1 server and it has started generating a handling exception when trying to access vaults for offline players. Here is a copy of the console log for reference. https://mclo.gs/PsJNkfU

**Comment by unisteven** [comment]
As I understand correctly
The steps to reproduce are:
- use /pv admin open {username}
Where username is currently offline? 

And this used to work on a older version of paper?

**Comment by ChiefWarPig** [comment]
Yes I was able to run the command before. Couldn't tell you wether it was the 1.20.6 or the 1.21 that broke it but it definitely worked before.
Yes to the reproduction steps it works fine for players that are online.
